### ISSUE TITLE
Android accessibility: when TalkBack is on, Gallery homescreen lists do not open

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#4750]
On Android, turn on TalkBack, then open the Gallery.

Expected: able to tap on the collapse lists (demos, components, etc), and see them selected and read to me.

Actual: Taps do not seem to be registered. The homescreen doesn't respond.

Turning off talkback, and going back into the homescreen of Flutter, and the UI elements respond.

**Comment by eseidelGoogle** [comment]
The lists are gone from the homescreen, however https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/two_level_list_demo.dart should show the same bug.

**Comment by goderbauer** [comment]
I cannot reproduce this in the two_level_list_demo.dart. Looks like it has been fixed. Feel free to re-open if you find a case where it's not working.
